infrastructure/aida-validation-batch-cf-template.json

```diff
@@ -0,0 +1,534 @@
+    "Outputs": {
+        "BatchComputeEnvironment": {
+            "Value": {
+                "Ref": "BatchComputeEnvironment"
+            }
+        },
+        "BatchSecurityGroup": {
+            "Value": {
+                "Ref": "BatchSecurityGroup"
+            }
+        },
+        "JobQueue": {
+            "Value": {
+                "Ref": "BatchJobQueue"
+            }
+        },
+        "JobDefinition": {
+            "Value": {
+                "Ref": "BatchJobDefinition"
+            }
+        },
+        "BatchInstanceRole": {
+            "Value": {
+                "Ref": "BatchInstanceRole"
+            }
+        },
+        "BatchServiceRole": {
+            "Value": {
+                "Ref": "BatchServiceRole"
+            }
+        },
+        "BatchTaskRole": {
+            "Value": {
+                "Ref": "BatchTaskRole"
+            }
+        }
+    },
+    "Parameters": {
+        "Release": {
+            "Type": "String",
+            "Description": "Release Identifier",
+            "Default": "develop"
+        },
+        "ComputeEnvironment": {
+            "Type": "String",
+            "Description": "Environment Name",
+            "Default": "aida-validation-cf-compute"
+        },
+        "ComputeEnvironmentMaxVCpus": {
+            "Type": "Number",
+            "Description": "The maximum number of vcpus for the compute environment.",
+            "Default": 5
+        },
+        "ComputeEnviornmentMinVCpus": {
+            "Type": "Number",
+            "Description": "The minimum number of vcpus for the compute environment.",
+            "Default": 0
+        },
+        "ComputeEnvironmentDesiredVCpus": {
+            "Type": "Number",
+            "Description": "The desired number of vcpus for the compute environment.",
+            "Default": 0
+        },
+        "ComputeNodes": {
+            "Type": "Number",
+            "Description": "The total number for nodes for the validation batch job.",
+            "Default": 2
+        },
+        "MainNodeRange": {
+            "Type": "String",
+            "Description": "The target node range for the main node (i.e. 0:0).",
+            "Default": "0:0"
+        },
+        "MainNodeMemory": {
+            "Type": "Number",
+            "Description": "The amount of memory for the main node (i.e. 2048).",
+            "Default": 2048
+        },
+        "MainNodeVpcus": {
+            "Type": "Number",
+            "Description": "The number of Vpcus for the main node (i.e. 2).",
+            "Default": 2
+        },
+        "WorkerNodeRange": {
+            "Type": "String",
+            "Description": "The target node range for the main node (i.e. 1:1).",
+            "Default": "1:1"
+        },
+        "WorkerNodeMemory": {
+            "Type": "Number",
+            "Description": "The amount of memory for the worker nodes (i.e. 2048).",
+            "Default": 2048
```

Comment:
I'm not sure if this ticket includes coming up with good default/parameters for the VMs, but this seems a bit small even for TA1.

Reply:
For testing I left it at the minimum. If you have a recommendation for any of these values let me know and I will update the defaults.

Reply:
Well, we haven't really gotten to trying to estimate worker VM sizes for the batch process (TA1 or TA2), so we'll hold off.  I agree to keep it at a minimum for testing.